src/index.js

```diff
@@ -24,6 +25,7 @@ ReactDOM.render(
   <Provider store={store}>
     <GlobalModal>
       <Router>
+        <TokenBar />
```

Comment:
This needs to be rendered under the `Router` provider so we can get the `location` and `history` objects from the `useLocation` and `useHistory` hooks